src/scrabble.js

```diff
@@ -1,5 +1,372 @@
-function scrabble() {
-  // write code here
+const onePoint = [
+  'A',
+  'a',
+  'E',
+  'e',
```

Comment:
Remove the upper or lower case characters and just one case, then run .toUpperCase or .toLowerCase on the input